### DISCUSSION TITLE
Is there an easy way to pip install insiders without having to specify the GH Token everytime ?

### DISCUSSION THREAD

**Discussion by UltraInstinct05** [squidfunk/mkdocs-material#3863] [Q&A] [answered]
Is there an easy way to upgrade versions of Insiders on a local project (using `pip install`) without having to specify the `GH_TOKEN` everytime ?

**Comment by squidfunk** [comment, marked as answer]
I guess you could fork the Insiders repository, and set up `git` over `ssh` if you haven't already. Next, test if you can `git clone` your private fork without entering credentials. If that works, you should be able to use `pip` in the following way:

```
pip install git+ssh://git@github.com/Ultra-Instinct-05/mkdocs-material-insiders.git
```

At least that's what [StackOverflow](https://stackoverflow.com/questions/4830856/is-it-possible-to-use-pip-to-install-a-package-from-a-private-github-repository) says. Unfortunately, it's not possible with the original repository, as I would have to add your public SSH keys to my account, which would grant you basically everything 😉 Thus, the `GH_TOKEN` or forking it is.

**Reply by squidfunk** [reply]
As an alternative, you might just set `GH_TOKEN` as an environment variable in your `.bashrc` (or whatever shell you're using).

**Reply by UltraInstinct05** [reply]
Ah, yes. I forgot I could just use an environment variable for this. Thanks !
